apps/bfd-pipeline/bfd-pipeline-app/src/main/java/gov/cms/bfd/pipeline/app/AppConfiguration.java

```diff
@@ -471,6 +463,17 @@ static ConfigLoader createConfigLoader(Function<String, String> getenv) {
     return LayeredConfiguration.createConfigLoader(DEFAULT_CONFIG_VALUES, getenv);
   }
 
+  /**
+   * Build a {@link ConfigLoader} for use in tests that just uses default values and those returned
+   * by the provided function.
+   *
+   * @param getenv function used to access environment variables (provided explicitly for testing)
+   * @return appropriately configured {@link ConfigLoader}
+   */
+  static ConfigLoader createConfigLoaderForTesting(Function<String, String> getenv) {
```

Comment:
Don't super love adding test-only code in app code if we can avoid it. Could we just call the builder with these values in the test where we need to set it up? (Or a shared test util if its used a lot?)

Reply:
We could and I considered doing that.  But then the provide default values map would need to be made public.  I felt that this revealed less private information and, honestly, this is only test only in the sense that it's only used by them.  Nothing about it depends on any test libraries or statics that are used only during testing.

I could just as easily have named it `createMinimalConfigLoader()`.